packages/e2e/package.json

```diff
@@ -32,7 +32,7 @@
     "eslint-plugin-import": "2.24.1",
     "eslint-plugin-prettier": "3.4.0",
     "fs-extra": "^7.0.1",
-    "mocha": "^9.2.0",
+    "mocha": "^10.0.0",
```

Comment:
Was this change required? Why?

Reply:
I'm actually ok with it, but I want to understand how this may affect existing users.

Reply:
I'm not necessarily sure if it was "required" per se, but mocha 10 is basically their version of the same upgrade we're doing here (i.e. deprecating node 12 support and adding node 18 support to their CI), so I felt like it made sense to include it.

Reply:
Makes sense